README.md

```diff
@@ -48,6 +48,24 @@ First `fuse` must be installed.
 
 Then clone the sources by `git clone --recursive`, and execute `linux-build.sh`.
 
+### FreeBSD
```

Comment:
Please add `(unofficial)` since the port is outside my control

Reply:
Sure, done.